packages/idealo.design/src/ui/pages/BlogPage/ListView.jsx

```diff
@@ -5,9 +5,9 @@ import withStyles from 'isomorphic-style-loader/withStyles'
 import { fetchList, fetchPostsByCategorySlug } from './data';
 import { Link, useParams, useHistory } from "react-router-dom";
 
-
-
 import s from './Blogpage.module.scss';
+import draftToHtml from "draftjs-to-html";
+const { htmlToText } = require('html-to-text');
```

Comment:
Again, please don't mix in CommonJS module format as we're using Ecmascript module format and this might break the builds. Please make use of import/export.

Reply:
Okay 👌 we'll fix that!